testutils/mockhyperbahn/hyperbahn_test.go

```diff
@@ -118,3 +121,52 @@ func TestMockForwards(t *testing.T) {
 	require.NoError(t, err, "Call failed")
 	require.True(t, called, "Advertised server was not called")
 }
+
+func TestMockIgnoresDown(t *testing.T) {
+	mockHB, err := mockhyperbahn.New()
+	require.NoError(t, err, "Failed to set up mock hyperbahm")
+
+	var (
+		moe1Called atomic.Bool
+		moe2Called atomic.Bool
+	)
+
+	moe1 := newAdvertisedEchoServer(t, "moe", mockHB, func() { moe1Called.Store(true) })
```

Comment:
I just want to make sure load is dsitributed, it won't be 100% even because the peer selection is kind of random, so we can't assert any specific numbers